common/src/leap/soledad/common/couch.py

```diff
@@ -59,9 +59,12 @@
 from u1db.remote.server_state import ServerState
 
 
-from leap.soledad.common import ddocs, errors
+from leap.soledad.common import ddocs
+from leap.soledad.common.errors import raise_server_error
```

Comment:
same as the other comment: this is couch specific, so maybe move to common.couch.errrors